app/src/main/java/app/MainViewModel.kt

```diff
@@ -22,6 +24,11 @@ class MainViewModel (var promptService : IPromptService) : ViewModel(){
 
     //Fetch Prompts
     //Get prompt from array list and present it to the user
+    fun fetchPrompts()  {
+        viewModelScope.launch   {
+            promptService.fetchPrompts();
+        }
+    }
```

Comment:
Added a function to call function from promptService to bring prompts to main view model to be displayed to the player.